core/client/fs/src/main/java/alluxio/client/block/stream/NettyPacketWriter.java

```diff
@@ -256,6 +268,14 @@ public void close() throws IOException {
       if (mChannel.isOpen()) {
         mChannel.pipeline().removeLast();
       }
+      if (closeFuture != null) {
```

Comment:
It does not matter. isOpen/removeLast are threadsafe. Done anyway since that is better.